package.json

```diff
@@ -167,7 +169,7 @@
     "lint:js:flow": "babel-node scripts/flow-check.js",
     "lint:js": "npm run lint:js:eslint && npm run lint:js:flow",
     "lint:css": "stylelint \"+(boilerplate|docs)/web_modules/**/*.css\"",
-    "lint": "npm run lint:js && npm run lint:css",
+    "lint": "npm-run-all --parallel lint:css lint:js:*",
```

Comment:
Sorry. I was wrong. It runs `lint:js && lint:css`. It treats the colon as sperator so you will need to do this: "npm-run-all lint:**"